requirements.txt

```diff
@@ -2,3 +2,4 @@ frappe
 unidecode
 pygithub
 googlemaps
+python-stdnum
```

Comment:
`python-stdnum` is not used anywhere in the code

Reply:
have a look at 

https://github.com/joezsweet/erpnext/blob/c95675c074408e6c522499960d910495784e3b7e/erpnext/stock/doctype/item/item.py#L486-L496

Reply:
it seems Item Barcode Doctype is missing ..

Reply:
have a look at 

https://github.com/joezsweet/erpnext/tree/multi-barcode/erpnext/stock/doctype/item_barcode

Reply:
ok